docs/framework/data/adonet/ef/language-reference/ref-entity-sql.md

```diff
@@ -8,7 +8,7 @@ Returns a reference to an entity instance.
   
 ## Syntax  
   
-```  
+```sql  
 REF( expression )   
 ```  
   
```

Comment:
these topics have an example section that is configured as C#. Wondering if they should be switched to SQL to get a better colorization?

Reply:
@mairaw, Yes. they're just sql statements that are referenced from csharp files. Some were changed to sql in #14760

Reply:
I implemented the suggestion from @mairaw